### PR TITLE
Backwards compatibility with previous version of ModLoader

### LINKED ISSUE #5: Retro compatibility with first version of ModLoader
Some modders have left the game and will probably never update their mods. So there are players who want to keep using their mods and therefore stay with the initial version of ModLoader.

I, as a modder, have therefore to handle two branches of my code. The tested one (with initial ModLoader), and the untested (hopefully working) ModLoader 0.2+ branch.

If retro compatibility is brought to ModLoader, modders will be able to focus developpment on only one version (the most recent) and players will be forced to update their ModLoader version without fear of having mods that don't work anymore.

### PR DESCRIPTION
**Pull request by pantolomin**
Backwards compatibility with previous version of ModLoader so that players can safely migrate to the newest version. Hopefully modders will follow.

### CODE REVIEW

File: PhoenixPointModLoader/LegacyPhoenixPointMod.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+
+namespace PhoenixPointModLoader
+{
+    class LegacyPhoenixPointMod : IPhoenixPointMod
```
Comment (Ijwu) [inline PhoenixPointModLoader/LegacyPhoenixPointMod.cs:7]: I like the idea of wrapping legacy mods with this class, clever! I understand it's nitpicking; please try to be consistent with  code conventions in the rest of the project. 

* Private fields should start with an underscore.
* Method names should be pascal case.

**Review by Ijwu — CHANGES_REQUESTED** [review]
Thanks a ton for this. I realize that my indescriminate obsolescence of legacy mods did not ring well with the community.

My only critical feedback is that the new `LegacyPhoenixPointMod` class should follow project conventions more closely. This isn't a PR killer, though, and if you're sick of me then I can do it--just let me know.

Thanks again for helping out! :)

**Comment by Ijwu** [comment]
P.S. sorry it took me so long to get to this. I've been unavailable for a while as I handle RL stuff.

**Comment by Ijwu** [comment]
I'll go ahead and merge this and make the changes myself. 

Thanks again for your contribution.

**Comment by pantolomin** [comment]
Oh, sorry. I hadn't seen that you wanted some modifications.
I'm a Java programmer so I don't know the writing conventions for C#. I learn some as I go.
I tried to apply those that I came across, but obviously there are a lot that I don't know.
It's really hard to go from one to an other.

**Comment by Ijwu** [comment]
@pantolomin I totally understand. It's very minor things and I just went ahead and did it because I found some time to do so. No worries. :)

**Pull request merged**